tools/js2c.cc

```diff
@@ -381,18 +360,45 @@ std::string GetVariableName(const std::string& id) {
   return result;
 }
 
-std::vector<std::string> GetCodeTable() {
-  size_t size = 1 << 16;
-  std::vector<std::string> code_table(size);
-  for (size_t i = 0; i < size; ++i) {
-    code_table[i] = std::to_string(i) + ',';
+// 382106 is the length of the string "0,1,2,3,...,65535,".
+// 65537 is 2**16 + 1
+// This function could be constexpr, but it might become too expensive to
+// compile.
+std::pair<std::array<char, 382106>, std::array<uint32_t, 65537>>
```

Comment:
I think this now needs a comment to explain the shape of the returned table (previously it was straightforward enough that a comment seemed redundant).